force-app/main/default/classes/SOQL.cls

```diff
@@ -50,9 +57,16 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Id> getContactIdsByLastName () {
+    //  The body of the method 
+    List<Id> contactIds = new List<Id>();
+        for (Contact c : [SELECT Id 
+                           FROM Contact 
+                           WHERE LastName = 'Smith' OR LastName = 'Jones']) {
```

Comment:
I like the SOQL for loop. I would still do vertical formatting for it as you've done above, with one slight modification:
```suggestion
        for (Contact c : [
                SELECT Id 
                FROM Contact 
                WHERE (LastName = 'Smith' OR LastName = 'Jones')
         ]) {
```

Reply:
Good stuff